raiden_contracts/data/source/services/OneToN.sol

```diff
@@ -94,6 +94,7 @@ contract OneToN is Utils {
         if (transferable > 0) {
             // register to avoid double claiming
             settled_sessions[_key] = expiration_block;
+            assert(expiration_block != 0);
```

Comment:
Can we somehow access the current block number? Otherwise I'm not sure if this check improves things. Maybe make it at least `expiration_block > 0`?

Reply:
I'll say `expiration_block > 0` here.  It reads much better.